### ISSUE TITLE
Enhance landscape table usability

### ISSUE THREAD

**Issue by leecalcote** [layer5io/layer5#65]
Enhance landscape table usability with:

1. sorting by descending alphabetical order on the leftmost column by default)
 - This would be the ability for a user to click on the column name and have the table sorted by that column.
-  The ability to click to switch between asc/desc
2. alternating colors in rows

**Comment by leecalcote** [comment]
@yogi4 the alternating colors you got into place are great. We could use a higher contrast font color on the table column headers and the ability to filter.

**Comment by Silvyre** [comment]
>
>
> @yogi4 the alternating colors you got into place are great. We could use a higher contrast font color on the table column headers and the ability to filter.

I would be happy to work on this!

**Comment by yogi4** [comment]
@Silvyre  Please feel free Rafi.

**Comment by Silvyre** [comment]
@yogi4 Thank you very much.

On another note, while I was looking at the Service Mesh Landscape page, I noticed that the last column of the Categories section were wrapping strangely (more apparent on smaller browser winodws) such that I had to scroll the window down before discovering that the rightmost category existed.

If you feel it's worthwhile, I would be happy to also look into improving this behaviour after opening an issue for it.
